files/0001-icingaweb2-module-audit-remote-ip-logging.patch

```diff
@@ -0,0 +1,67 @@
+From 3bf6f47910834e4ddd9c7fb4bf085daed5ab515a Mon Sep 17 00:00:00 2001
```

Comment:
If you're now using our own fork with the PR merged, the patch file is no longer needed, right?

Reply:
Removed.